src/v/archival/probe.h

```diff
@@ -29,7 +29,7 @@ namespace archival {
 /// The unit of measure is offset delta.
 class ntp_level_probe {
 public:
-    ntp_level_probe(per_ntp_metrics_disabled disabled, model::ntp ntp);
+    ntp_level_probe(per_ntp_metrics_disabled disabled, const model::ntp& ntp);
```

Comment:
I don't see the change to go from `_ntp` to `ntp` on the labels here? what am i missing.

Reply:
there is no `_ntp` anymore, no point to store the `ntp` instance here, it's only used to initialize labels

Reply:
let me read the .cc file

Reply:
got it. lgtm.